owncloudData/build.gradle

```diff
@@ -27,6 +27,8 @@ android {
         }
 
         multiDexEnabled true
+
+        manifestPlaceholders = [appAuthRedirectScheme: 'oc']
```

Comment:
Is this needed in data layer?

Reply:
Already explained in https://github.com/openid/AppAuth-Android/issues/325